riptide-core/src/test/java/org/zalando/riptide/RequesterTest.java

```diff
@@ -88,14 +88,15 @@ public void shouldEncodeInlinedQueryParams() {
 
     @Test
     public void shouldAppendQueryParams() {
-        server.expect(requestTo("https://api.example.com?foo=bar&foo=baz&bar=null"))
+        server.expect(requestTo("https://api.example.com?foo=true&foo=null&foo=bar&bar=123"))
```

Comment:
> With quotes? oO

No without :see_no_evil: but whatever we do here is probably unexpected from someone.